main.js

```diff
@@ -393,15 +393,15 @@ $(function() {
       return;
     }
 
-    //if player selected "buy all", count is set to highest purchesable amount
-    const all = towns[currentTown].stockpiles[productIndex];
+    //if player selected "buy all, count is set to highest purchesable amount
```

Comment:
This is a good comment, but it should be 2 lines lower (where you actually set `count`), and the typos fixed:
```
// if player selected "buy all", count is set to highest purchasable amount
```